bids/layout/tests/test_layout.py

```diff
@@ -48,6 +48,13 @@ def test_index_metadata(index_metadata, query, result, mock_config):
     assert metadata.get('RepetitionTime') == result
 
 
+
+def test_config_filename(config_filename):
+    layout = BIDSLayout(get_test_data_path(),indexer=BIDSLayoutIndexer(config_filename=get_test_data_path()+'/../'+'bids_specs_with_oligarchy.json'),validate=False)
+    # make sure that the funny custom config that has every instance of 'session' renamed to 'oligarchy' is respected.
+    assert 'oligarchy' layout.get_entities().keys()
```

Comment:
```suggestion
    assert 'oligarchy' in layout.get_entities()
```

Reply:
I was going to say, if it was going to be a headache, I could just comment this line out. On top of my late night typo, this doesn't seem to have worked:

https://github.com/bids-standard/pybids/runs/4244354483?check_suite_focus=true#step:10:521

> >       assert 'oligarchy' in layout.get_entities()
> E       AssertionError: assert 'oligarchy' in {'acquisition': <bids.layout.models.Entity object at 0x7f89a841c8e0>, 'ceagent': <bids.layout.models.Entity object at ...layout.models.Entity object at 0x7f89a8217730>, 'direction': <bids.layout.models.Entity object at 0x7f89a841c040>, ...}

It sort of seems like `config_filename` isn't actually affecting the schema that gets defined. Did it maybe get hard-coded into pybids at some point between 0.14 and 0.13?